workflows/prognostic_c48_run/runtime/names.py

```diff
@@ -35,6 +35,7 @@
     "override_for_time_adjusted_total_sky_net_shortwave_flux_at_surface",
     "override_for_time_adjusted_total_sky_downward_longwave_flux_at_surface",
     "ocean_surface_temperature",
+    "surface_temperature",
```

Comment:
Unrelated to this PR but I noticed that surface temp was not included in the prephysics updated quantities (should be as it is basically the same update as ocean temp)